.github/workflows/default-windows-2019.yml

```diff
@@ -0,0 +1,31 @@
+name: Build PR (Windows Server 2019)
+on:
+  workflow_dispatch:
```

Comment:
Do we keep this `workflow_dispatch`? It allows us to manually trigger the workflow. I kept here to run tests.

Reply:
I don't think there is ever going to be a situation where we need to run this with `workflow_dispatch`

Reply:
I'm going to remove after everyone's approval. Just so I can trigger workflows if I need to test and change request.

Reply:
There's also no downside to just keeping it, other workflows have it